roles/galaxy.movedata/tasks/main.yml

```diff
@@ -19,24 +19,8 @@
 - include: export.yml
   when: export_dir_exists.stat.exists and not export_exists.stat.exists
 
-- name: Adjust proxy prefix in galaxy.ini
-  replace: dest={{ galaxy_config_dir }}/galaxy.ini regexp='(.*)prefix = (?!{{ nginx_galaxy_location }}).*' replace='\1prefix = {{ nginx_galaxy_location }}' backup=yes
-  tags: galaxy_config_files
-
-- name: Get host's external IP address
-  shell: curl --silent --fail http://icanhazip.com
-  register: host_external_ip
-
-- name: Adjust ftp_upload_site IP address in galaxy.ini
```

Comment:
Same here, this is so that the correct address for FTP is being displayed (which is less important now that you can also upload huge files via the browser, but still users may want to use this ?)

Reply:
Same answer ;-) I dont't want to drop it (as for miniconda role), I want to better organise.
Sorry for mixing everything in the PR: I am re-activating the problems as I check codes...